### ISSUE TITLE
Create a github.io page to download builds from

### ISSUE THREAD

**Issue by suhr** [qTox/qTox#564]
While Github is developer-friendly, it is not very _user-friendly._

**Comment by agilob** [comment]
there are already 3 different, independent pages to download builds from...

**Comment by suhr** [comment]
Which? I know only about github and tox.im wiki (which redirect you to github)...

**Comment by anoadragon453** [comment]
This is fixed with https://tox.chat/download.html I assume.

**Comment by zetok** [comment]
@anoadragon453 

> This is fixed with https://tox.chat/download.html I assume.

'no'. This crap is not the qTox page, and it doesn't have correct instructions.

There is https://qtox.github.io/, but it doesn't really address this issue either.

**Comment by anoadragon453** [comment]
Does it have to be qTox specific though? And if the instructions are incorrect, create an issue on [Tox/tox.chat](https://github.com/Tox/tox.chat/issues) ?

**Comment by zetok** [comment]
@anoadragon453 people at tox.chat are well aware of their instructions being wrong. As for me creating issues, making them for tox.chat would be a waste of my time.

**Comment by anoadragon453** [comment]
@zetok Fair enough.

**Comment by anthonybilinski** [comment]
Is this still an issue? As far as I can tell https://tox.chat/download.html now has accurate instructions linking to our github binaries + flatpak and appImage. https://qtox.github.io/ links to same downloads. Is there something more wanted?

**Comment by zetok** [comment]
>https://qtox.github.io/ links to same downloads. Is there something more wanted?

Not quite, https://qtox.github.io links to the qTox's README.md, which is arguably not as user-friendly as simple webpage for english speaking people, and is utterly not friendly for anyone else who doesn't speak english.

What is wanted:

- simple page with links to downloads, no unnecessary/distracting elements
- translatable download page

[Framework](https://github.com/qTox/qTox-Website) is already there, from what I understand one would need to create e.g. `download.mustache` (analogous to `index.mustache`), and `download.en.json` with translatable strings. Alternatively, just the `download.mustache` add new translatable strings to `index.en.json` and then hack around the `buildsite.py` to make it work. The latter option is preferable, as it would make it easier/straightforward to make it work with weblate and avoid problem with running into inconsistency issues with availability of localized download pages.
